garden_ai/app/notebook.py

```diff
@@ -137,7 +137,61 @@ def handler(signal, frame):
     return
 
 
-@notebook_app.command()
+@notebook_app.command(no_args_is_help=True)
+def debug(
+    path: Path = typer.Argument(
+        ...,
+        file_okay=True,
+        dir_okay=False,
+        writable=True,
+        readable=True,
+        help=(
+            "Path to a .ipynb notebook whose remote environment will be approximated for debugging."
+        ),
+    )
+):
+    """Open the debugging notebook in a pre-prepared container.
+
+    Changes to the notebook file will NOT persist after the container shuts down.
+    Quit the process with Ctrl-C or by shutting down jupyter from the browser.
+    """
+    docker_client = docker.from_env()
+    base_image = _get_notebook_base_image(path) or "gardenai/test:latest"
+    image = build_notebook_session_image(
+        docker_client, path, base_image, print_logs=False
+    )
+    if image is None:
+        typer.echo("Failed to build image.")
+        raise typer.Exit(1)
+    image_name = str(image.tags[0])  # str used to guarantee type-check
```

Comment:
best way to tweak it might be to give `start_container_with_notebook` a `pull=True` kwarg and call it like 
`container = start_container_with_notebook(..., base_image=image.id, mount=False, pull=False)`, since images always have an `image.id` even if they're not yet tagged